react/features/base/conference/middleware.js

```diff
@@ -643,3 +647,27 @@ function _trackAddedOrRemoved(store, next, action) {
 
     return next(action);
 }
+
+/**
+ * Updates the conference object when the local participant is updated.
+ *
+ * @param {Store} store - The redux store in which the specified {@code action}
+ * is being dispatched.
+ * @param {Dispatch} next - The redux {@code dispatch} function to dispatch the
+ * specified {@code action} to the specified {@code store}.
+ * @param {Action} action - The redux action which is being dispatched in the
+ * specified {@code store}.
+ * @private
+ * @returns {Object} The value returned by {@code next(action)}.
+ */
+function _updateLocalParticipantInConference({ getState }, next, action) {
+    const { conference } = getState()['features/base/conference'];
+    const { participant } = action;
+    const result = next(action);
+
+    if (conference && participant.name) {
+        conference.setDisplayName(participant.name);
```

Comment:
Is it guaranteed this will fire for the local participant only? If a remote participant does a name change, will this not get fired?

Reply:
I fixed this bug and relocated the commit to a separate PR.